src/platform.js

```diff
@@ -3,6 +3,22 @@ class Platform {
         Platform.OS = platform.name();
         Platform.Version = platform.version();
     }
+
+    static setLinking(linking) {
+      Platform.linking = linking;
+    }
+
+    static canOpenUrl(url) {
+      try {
+        return Platform.linking.canOpenURL(url);
+      } catch {
+        return new Promise((resolve, reject) => reject({ message: 'Invalid format.' }));
+      }
+    }
+
+    static openUrl(url) {
```

Comment:
same here?

Reply:
I think we don't need `async` here. Check [this](https://javascript.info/async-await).